tap_zendesk/schemas/ticket_forms.json

```diff
@@ -0,0 +1,157 @@
+{
+  "properties": {
+    "created_at": {
+      "type": [
+        "null",
+        "string"
+      ],
+      "format": "date-time"
+    },
+    "name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "display_name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "raw_display_name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "position": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "raw_name": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "custom_field_options": {
```

Comment:
Looks like this is in our `ticket_fields` schema, might be a good idea to make it a subschema reference so that we can share the properties, in case it gets changed? Looks like they're the same right now, so no reason to make it here, but thought I'd mention :)

Reply:
Actually, this isn't in the schema on the API docs, should probably be removed if we can't get it. https://developer.zendesk.com/rest_api/docs/core/ticket_forms